mtags/src/main/scala-3/scala/meta/internal/pc/completions/KeywordsCompletions.scala

```diff
@@ -23,6 +22,8 @@ object KeywordsCompletions:
           case kw if (kw.isPackage || kw.isTemplate) && notInComment =>
             CompletionValue.keyword(kw.name, kw.insertText)
         }
+      case Select(qual, name) :: _ if "match".startsWith(name.toString()) =>
+        List(CompletionValue.keyword("match", "match\n\tcase $0\n"))
```

Comment:
Will `\t` be valid for different indentation sizes and styles (spaces vs tabs)?

Reply:
That's actually an indication for the editor to increase the indentation. It already works this way in other places.